File: Conditionals/OldEnough.py
#Evelyn Chennault, Old Enough Python

print("Hello! I will determine if you can legally have certain privileges in the USA.")
num = int(input("How old are you?\n"))
print("\n")

#If you can vote
if num >= 5 and num <= 11:
    print("You can not vote, get a learners permit, or drive, but you can go to school.")
elif num >= 12 and num <= 14:
    print("You can not vote or drive, but you can get a learners permit and go to school.")
elif num >= 15 and num <= 17:
    print("You can not vote, but you can get a learners permit, drive and go to school.")
elif num >= 18:
    print("You can vote, get a learners permit, drive, and go to school.")
else: 
    print("You can not vote, get a learners permit, drive, or go to school.")

File: Conditionals/OldEnough.c
//Evelyn Chennault, Old Enough C
#include <stdio.h>
#include <string.h>

int main(void){
    int num;
    printf("This prgram will tell you if you can vote, get a permit, dirve, and go to school.");
    printf("Please type your age here:\n");
    scanf("%d", &num);
    if(num <= 5 && num >= 11){
        printf("You can not vote, get a learners permit, or drive, but you can go to school.");
    }else if(num >= 12 && num <= 14){
        printf("You can not vote or drive, but you can get a learners permit and go to school.");
    }else if (num >= 15 && num <= 17){
        printf("You can not vote, but you can get a learners permit, drive and go to school.");
    }else if(num >= 18){
        printf("You can  vote, get a learners permit, drive and go to school.");
    }
}